tests/Test/AbstractFixerTestCase.php

```diff
@@ -494,7 +500,11 @@ private function getLinter(): LinterInterface
     private static function assertValidDescription(string $fixerName, string $descriptionType, string $description): void
     {
         self::assertMatchesRegularExpression('/^[A-Z`].+\.$/s', $description, sprintf('[%s] The %s must start with capital letter or a ` and end with dot.', $fixerName, $descriptionType));
-        self::assertStringNotContainsString('phpdocs', $description, sprintf('[%s] `PHPDoc` must not be in the plural in %s.', $fixerName, $descriptionType));
+        self::assertSame(
+            0,
+            substr_count($description, 'phpdocs') - substr_count($description, '`phpdocs_'), // allow the option to have "`phpdocs_*`"
```

Comment:
I think right now we have _only_ `phpdocs_` (plural) as rule option, is it the case?